apps/src/code-studio/components/progress/teacherPanel/StudentTable.jsx

```diff
@@ -87,7 +87,7 @@ class StudentTable extends React.Component {
                   <div style={styles.name}>
                     {student.name}
                     {!!DCDO.get('family-name-features', false) &&
-                      ` ${student.familyName}`}
+                      ` ${student.familyName || ''}`}
```

Comment:
Ahh I guess it makes sense that this was happening. Works for me

Reply:
ugh, i wonder if i broke it when i put it behind the flag. thanks for fixing!